Sources/PackageLoading/ToolsVersionLoader.swift

```diff
@@ -45,6 +45,16 @@ public class ToolsVersionLoader: ToolsVersionLoaderProtocol {
 
         // Get the version specifier string from tools version file.
         guard let versionSpecifier = ToolsVersionLoader.split(contents).versionSpecifier else {
+            // Try to diagnose if there is a misspelling of the swift-tools-version comment.
+            let splitted = contents.contents.split(separator: UInt8(ascii: "\n"), maxSplits: 1, omittingEmptySubsequences: false)
+            let misspellings = [
+                "swift-tool", "tool-version",
+            ]
+            if let firstLine = ByteString(splitted[0]).asString,
```

Comment:
Yep it will always have at least one value because omittingEmptySubsequences is false